components/raftstore/src/coprocessor/config.rs

```diff
@@ -50,6 +50,8 @@ pub struct Config {
     pub region_bucket_size: ReadableSize,
     // region size threshold for using approximate size instead of scan
     pub region_size_threshold_for_approximate: ReadableSize,
+    #[online_config(skip)]
+    pub prefer_gen_bucket_by_approximate: bool,
```

Comment:
I don't think we need a switch here if the performance was verified.

Reply:
> I don't think we need a switch here if the performance was verified.

By adding an undocumented config it's useful for performance troubleshooting in the future to verify if imbalanced buckets play a role in some scenario.  We need a way to generate absolutely accurate buckets and this flag can help do that.